governance/polls/templates/Atlas/Atlas Edit Weekly Cycle Proposal - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Atlas Edit Weekly Cycle Proposal - $date_MONTH_DD,_YYYY
+summary: $sentence_summary
+discussion_link: $discussion_link
+parameters:
+    input_format:
+        type: single-choice
+        abstain: [0]
+    victory_conditions:
+        - {
+            type: 'and',
+            conditions: [
+                { type : plurality },
+                { type : comparison, comparator : '>=', value: 20000 }
+            ]
+        }
+        - {type : default, value : 2 }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: $YYYY-MM-DDT16:00:00
+end_date: $YYYY-MM-DDT16:00:00
+---
+# Atlas Edit Weekly Cycle Proposal - $date_MONTH_DD,_YYYY
+
+The $poll_deployers have placed an [Atlas Edit Weekly Cycle Proposal](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59|0db30308) into the [voting system](https://vote.makerdao.com/polling) [on behalf of Prime Delegate $Delegate_Name]($link). This Governance [Poll](https://sky-atlas.powerhouse.io/#A.1.9.2_Atlas_Edit_Weekly_Cycle-4a8ad9ad-5c5d-4994-9b46-f04c0e61ce59%7C0db30308) will be active for three days beginning on $date_DAY,_MONTH_DD at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community may vote in this poll to express support or opposition to the following Atlas Edit Weekly Cycle Proposal:
+
+- [Atlas Edit Pull Request]($Pull_Request_link)
+- [Proposal Discussion Thread]($discussion_link)
+
+A brief summary of this Atlas Edit has been provided by the Author and is shown below:
+
+*$paragraph_summary*
+
+## Outcomes
+
+This poll implements a **Minimum Positive Participation** value. The Minimum Positive Participation for Atlas Edit Weekly Cycle Proposals is currently set to **20,000 MKR**.
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option AND the votes for the 'Yes' option exceed 20,000 MKR, then the following actions will be taken:**
```

Comment:
[Votes need not > 20,000 but >=](https://sky-atlas.powerhouse.io/#A.1.9.2.4_Minimum_Positive_Participation-619a6715-ae9e-4b83-984a-0bc833cefd01|0db30308ce59)

(This is different from the Monthly Cycle, where the minimum positive participation must be exceeded).

Reply:
The monthly cycle is also "equal or exceed" now we have pushed updates to both templates